### DISCUSSION TITLE
Update 2.0

### DISCUSSION THREAD

**Discussion by DIMFLIX** [themix-project/themix-gui#546] [Q&A]
When is the 2.0 update scheduled to be released?
How much will the interaction with scripts, in particular with the base16 plugin, change?
I'm really looking forward to the centralization of plugins in oomox_cli ❤️

**Comment by actionless** [comment]
there is really no schedule, you could potentially speed the progress up by contributing the code (https://github.com/themix-project/themix-gui/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) or making a donation

"Multi-Export CLI" is the rework of old, separate per-plugin CLIs.
The full "centralization" would be achieved when Import Plugins would be also supported in CLI, not only export ones.
